### PR TITLE
feature/NRLF-143-producer-postman-collection

### PR DESCRIPTION
**Pull request by Rohoolio**
Uploaded producer postman collection

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by brianbayes — DISMISSED** [review]

File: postman/national_record_locator_producer_sandbox.json [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+				"disabledSystemHeaders": {
+					"accept": true
+				}
+			},
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "nhsd-correlation-id",
+						"value": "nrl-consumer-api-sandbox",
```
Comment (deepaattiguppe) [inline postman/national_record_locator_producer_sandbox.json:22]: change the value to 'nrl-producer-api-sandbox' throughout the file

File: postman/national_record_locator_producer_sandbox.json [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+						"type": "text"
+					},
+					{
+						"key": "accept",
+						"value": "version=1.0",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
```
Comment (deepaattiguppe) [inline postman/national_record_locator_producer_sandbox.json:52]: change the mode to 'json' where a request body is being passed through the file

File: postman/national_record_locator_producer_sandbox.json [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+						"nrl-producer-api",
+						"FHIR",
+						"R4",
+						"DocumentReference",
+						"RY26A-1234567893"
+					]
+				}
+			},
+			"response": []
+		}
```
Comment (deepaattiguppe) [inline postman/national_record_locator_producer_sandbox.json:396]: add variable 'base_url' 	"variable": [
		{
			"key": "base_url",
			"value": "https://sandbox.api.service.nhs.uk/nrl-producer-api",
			"type": "string"
		}

**Review by deepaattiguppe — CHANGES_REQUESTED** [review]

File: postman/national_record_locator_producer_sandbox.json (hunk 1 above)
Comment (shahmed1) [reply postman/national_record_locator_producer_sandbox.json]: updated

File: postman/national_record_locator_producer_sandbox.json (hunk 2 above)
Comment (shahmed1) [reply postman/national_record_locator_producer_sandbox.json:52]: "language": "json" added.

File: postman/national_record_locator_producer_sandbox.json (hunk 3 above)
Comment (shahmed1) [reply postman/national_record_locator_producer_sandbox.json:371]: base_url variable added.

**Review by deepaattiguppe — APPROVED** [review]

**Review by deepaattiguppe — APPROVED** [review]

**Review by deepaattiguppe — APPROVED** [review]

**Review by deepaattiguppe — APPROVED** [review]

**Review by brianbayes — APPROVED** [review]

**Pull request merged**